src/main/java/pb360/data/entity/MessageObject.java

```diff
@@ -0,0 +1,27 @@
+package pb360.data.entity;
+
+import java.util.Date;
+
+import org.springframework.data.annotation.Id;
+
+import com.mongodb.client.MongoDriverInformation.Builder;
+
+public final class MessageObject {
+	static final int MAX_LENGTH_DESCRIPTION = 500;
+	static final int MAX_LENGTH_TITLE = 100;
```

Comment:
please remove 2 variables, seen it's not used anyplace

Reply:
I understand, those two only appear in Controller file, so i will remove it from MessageObject